Sources/ClientRuntime/Networking/Streaming/FileStream.swift

```diff
@@ -70,10 +81,21 @@ class FileStream: Stream {
         }
     }
 
+    func readToEndAsync() async throws -> Data? {
```

Comment:
Same as `readAsync(upToCount)` above, blocking file operation is wrapped in a continuation.